### ISSUE TITLE
[Bug Report]Switch 标签 name写错了？

### ISSUE THREAD

**Issue by L-Angel** [view-design/ViewUI#248]
### Environment
macos chrome vue(3.10.0)

### Reproduction link
[https://github.com/view-design/ViewUI/blob/master/src/components/switch/switch.vue](https://github.com/view-design/ViewUI/blob/master/src/components/switch/switch.vue)

### Steps to reproduce
Switch标签不生效 iSwitch生效

### What is expected?
Switch标签正常显示

### What is actually happening?
Switch没有被渲染

**Comment by Sway007** [comment]
yeah, component name is iSwitch according to https://github.com/view-design/ViewUI/blob/master/src/components/switch/switch.vue, please fix official docs

**Comment by David-Sam** [comment]
用<i-switch></i-switch>写法，<Switch></Switch>写法不支持

**Comment by David-Sam** [comment]
`<i-switch></i-switch>`

**Comment by David-Sam** [comment]
采用这种`<i-switch></i-switch>`
